osu.Game/Graphics/UserInterface/OsuTabControl.cs

```diff
@@ -25,15 +25,15 @@ public class OsuTabControl<T> : TabControl<T>
 
         protected override bool InternalContains(Vector2 screenSpacePos) => base.InternalContains(screenSpacePos) || Dropdown.Contains(screenSpacePos);
 
+        private bool isEnumType => typeof(T).IsEnum;

```

Comment:
can be static if it's private(means you won't get an instance to call it from outside)